tests/unit/subscriptions/test-class-wc-payments-product-service.php

```diff
@@ -274,9 +274,7 @@ public function test_get_wcpay_product_id() {
 		$this->assertSame( $mock_product_id, $this->product_service->get_wcpay_product_id( $this->mock_product ) );
 	}
 
-	/**
```

Comment:
Looks like this got removed accidentally?

Reply:
Thanks for catching that! . I have added it back - https://github.com/Automattic/woocommerce-payments/pull/4496/commits/4ef7523e175471d595ff3fc949c865ebbdd04425